source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+    Attempting to renew your proxy certificate...Proxy certificate has 720:00:00  (30.0 days) left.
 
-The data transfer nodes are assigned to a site specific partition on
-the **es cluster**. Use the following command to view current and/or
-available partitions:
+            Welcome to gaea.rdhpcs.noaa.gov
+    Gateway to gaea-c5.ncrc.gov and other points beyond
 
- .. code-block:: shell
+    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    !! RDHPCS Policy states that all user login sessions shall be terminated     !!
+    !! after a maximum duration of seven (7) days. ALL user login sessions will  !!
+    !! be dropped from the Princeton Bastions at 4AM ET / 2AM MT each Monday     !!
+    !! morning, regardless of the duration. Please note: This will NOT impact    !!
+    !! batch jobs, cron scripts, screen sessions, remote desktop, or data        !!
+    !! transfers.                                                                !!
+    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-     $ scontrol show partitions
+    Hostname            Description
+    gaea                C5 head nodes
+    gaea51              C5 head node
+    gaea52              C5 head node
+    gaea53              C5 head node
+    gaea54              C5 head node
+    gaea55              C5 head node
+    gaea56              C5 head node
+    gaea57              C5 head node
+    gaea58              C5 head node
+    gaea60              T6 Test access only
+    gaea61              C6 head node
+    gaea62              C6 head node
+    gaea63              C6 head node
+    gaea64              C6 head node
+    gaea65              C6 head node
+    gaea66              C6 head node
+    gaea67              C6 head node
+    gaea68              C6 head node
 
-     or
+    You will now be connected to NOAA RDHPCS: Gaea (CMRS/NCRC) C5 system.
+    To select a specific host, hit ^C within 5 seconds.
+    ^CEnter a hostname, or a unique portion of a hostname []:
 
-  .. code-block:: shell
+****************
+Data and storage
+****************
 
-     $ scontrol show partitions | grep dtn
+NFS filesystems
+===============
 
+Users and projects are given space on the :abbr:`NFS (Network File System)`.
+These locations are ideal for storing user and project applications,
+executables, and small data files.
 
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Area
+      - Path
+      - Permissions
+      - Quota
+      - Backups
+      - Purged
+      - On Compute Nodes
+    * - User Home
+      - :file:`/ncrc/home[12]/<userID>`
+      - User set
+      - 50 GB
+      - Yes
+      - No
+      - Yes
+    * - Project Home
+      - :file:`/ncrc/proj/<projID>`
+      - Project set
+      - 100 GB
+      - Yes
+      - No
+      - Yes
+
+
+GPFS filesystems
+================
+
+Each compute cluster, C5 and C6, have their own file system called F5 and F6
```

Comment:
Recommend:

Each compute cluster has its own file system

Reply:
Updated in. anew push.